ppr-ui/src/components/search/SearchBarList.vue

```diff
@@ -0,0 +1,194 @@
+          <v-list-item-content style="padding: 9px 0;">
+            <v-row
+              :id="`srch-type-drop-${item.group}`"
+              style="width: 45rem; pointer-events: all;"
+              @click="toggleGroup(item.group)"
+            >
+              <v-col class="py-0" align-self="center" cols="11">
+                <span class="search-list-header"><v-icon class="menu-icon">{{item.icon}}</v-icon>
+                {{ item.textLabel }}</span>
+              </v-col>
+              <v-col class="py-0" align-self="center" cols="auto">
+                <v-btn icon small style="pointer-events: all;">
+                  <v-icon v-if="displayGroup[item.group]" class="expand-icon" color="primary">mdi-chevron-up</v-icon>
+                  <v-icon v-else class="expand-icon" color="primary">mdi-chevron-down</v-icon>
+                </v-btn>
+              </v-col>
+            </v-row>
+          </v-list-item-content>
+        </template>
+        <template v-else class="search-list">
+          <v-list-item
+            v-if="displayGroup[item.group]"
+            class="copy-normal"
+            @click="selectSearchType(item)"
+          >
+            <v-list-item-title>
+              {{ item.searchTypeUI }}
+            </v-list-item-title>
+          </v-list-item>
+        </template>
+      </template>
+  </v-select>
+</template>
+<script lang="ts">
+import { defineComponent, reactive, toRefs, computed, ref } from '@vue/composition-api'
+import { useGetters } from 'vuex-composition-helpers'
+import { MHRSearchTypes, SearchTypes } from '@/resources'
+import { UISearchTypes, APISearchTypes } from '@/enums'
+import { SearchTypeIF } from '@/interfaces' // eslint-disable-line no-unused-vars
+import { getFeatureFlag } from '@/utils'
+
+export default defineComponent({
+  name: 'SearchBarList',
+  emits: ['selected'],
+  props: {
+    defaultSelectedSearchType: {
+      type: Object as () => SearchTypeIF
+    },
+    defaultCategoryMessage: {
+      type: String,
+      default: ''
+    }
+  },
+  setup (props, { emit }) {
+    const {
+      isRoleStaffReg,
+      hasPprRole,
+      hasMhrRole
+    } = useGetters<any>([
+      'isRoleStaffReg',
+      'hasPprRole',
+      'hasMhrRole'
+    ])
+    const searchSelect = ref(null)
+    const localState = reactive({
+      searchTypes: UISearchTypes,
+      searchTypeValues: APISearchTypes,
+      selectedSearchType: props.defaultSelectedSearchType,
+      categoryMessage: computed((): string => {
+        return props.defaultCategoryMessage
+      }),
+      searchTypeLabel: computed((): string => {
+        if (localState.selectedSearchType?.searchTypeUI) {
+          return localState.selectedSearchType.searchTypeUI
+        }
+        return 'Select a search category'
+      }),
+      origItems: computed((): Array<SearchTypeIF> => {
+        const allSearchTypes = []
+        if (isRoleStaffReg.value) {
+          if (getFeatureFlag('bcregistry-ui-ppr-mhr-staff-only')) {
```

Comment:
hey steph this feature flag should be something like 'mhr_enabled' and then in launch darkly we specify who has access (staff, specific staff, specific users, etc.). This way we can turn off MHR for users at any time too while keeping ppr open

Reply:
I updated the feature flag name